client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,94 @@
+# KServe API usage samples
+
+OpenVINO Model Server 2022.2 release introduced support for [KServe API](https://github.com/kserve/kserve/tree/master/docs/predict-api/v2).
+
+## Before you run the samples
+
+### Clone OpenVINO&trade; Model Server GitHub repository and go to the directory with the samples.
+```Bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/client/cpp/kserve-api
+```
+
+### Build client library and samples
+```Bash
+cmake . && make
+```
+
+## GRPC Examples <a name="grpc-api"></a>
+
+### Run the Client to get server liveness <a name="grpc-server-live"></a>
+
+- Command
+
+```Bash
+./grpc_server_live --help
+Sends requests via KServe gRPC API to check if server is alive.
+Usage:
+  grpc_infer_dummy [OPTION...]
+
+  -h, --help              Show this help message and exit
+      --grpc_address arg  Specify url to grpc service.  (default: 
+                          localhost)
+      --grpc_port arg     Specify port to grpc service.  (default: 9000)
+      --timeout arg       Request timeout. (default: 0)
+```
+
+- Usage Example 
+
+```Bash
+./grpc_server_live --grpc_port 9000 --grpc_address localhost
+Server Live: True
+```
+
+## GRPC Examples with Dummy Model
+
+This section demonstrates request to simple model, which increments each provided value. 
+
+### Start the Model Server Container with Dummy Model
+```Bash
+docker run --rm -d -v $(pwd)/src/test/dummy:/models -p 9000:9000 openvino/model_server:latest --model_name dummy --model_path /models --port 9000 
```

Comment:
`$pwd` is expected to be a specific value here as this command will work properly only when run from top level directory. Currently validation will not be able to handle that part. 
@ngrozae do you have any recommendation on how to handle this?

Reply:
Also, let's move this part before liveness check sample is described, so we have a consistent flow:
`building lib and samples -> starting OVMS -> checking liveness -> running inference`

Reply:
To use ${pwd} here we should be in model_server directory. So you can either use $(pwd)../../../src/test/dummy here or change order of commands to:
git clone https://github.com/openvinotoolkit/model_server.git
cd model_server
docker run --rm -d -v $(pwd)/src/test/dummy:/models -p 9000:9000 openvino/model_server:latest --model_name dummy --model_path /models --port 9000 
cd client/cpp/kserve-api
cmake . && make